docs/Installing.asciidoc

```diff
@@ -308,12 +308,14 @@ For a local instance setup you can simply execute the script:
 This will automatically setup a local apache http proxy. Read on for more
 detailed setup instructions with all the details.
 
+If you wish to run openQA behind an http proxy (apache, NGINX, etc..) then see the
```

Comment:
```suggestion
If you wish to run openQA behind an HTTP proxy (Apache, NGINX, …) then see the
```
("etc" is usually just followed by one dot. However, I suggest you simply use "…" instead of "etc".)